harness/src/network.rs

```diff
@@ -57,6 +57,8 @@ pub struct Network {
     pub storage: HashMap<u64, MemStorage>,
     /// Drop messages from `from` to `to` at a rate of `f64`.
     dropm: HashMap<Connection, f64>,
+    /// Delay sending messages from `from` to `to` at a rate of `f64` by blocking `u64` microseconds.
+    delaym: HashMap<Connection, (f64, u64)>,
```

Comment:
Do you think it's worth using https://doc.rust-lang.org/std/time/struct.Duration.html here?

Reply:
It should be a Duration. Thanks!